sklearn/neighbors/tests/test_nearest_centroid.py

```diff
@@ -97,6 +97,20 @@ def test_pickle():
                        " after pickling (classification).")
 
 
+def test_shrinkage():
+    # Ensure that the shrinking is correct.
+    # The corrent result is calculated by R (pamr),
+    # which is implemented by the author of the original paper.
+    # (One need to modify the code to output the new centroid in pamr.predict)
+
+    X = np.array([[0, 1], [1, 0], [1, 1], [2, 0], [6, 8]])
+    y = np.array([1, 1, 2, 2, 2])
+    clf = NearestCentroid(shrink_threshold=0.1)
```

Comment:
@jnothman Thanks, but do you think it is necessary to test another threshold? Frankly I may not think so(and I have tried several cases on my PC). Or I may misunderstand your meaning.